### PR TITLE
Standardize the easy_infra interface (resolves #29)

### LINKED ISSUE #29: Ensure a consistent interface
https://github.com/docker-library/official-images#consistency

### PR DESCRIPTION
**Pull request by JonZeolla**
# Contributor Comments
This standardizes the `easy_infra` interface in line with [Docker recommendations](https://github.com/docker-library/official-images#consistency).  When testing, ensure you run `make build` first.

## Pull Request Checklist
Thank you for submitting a contribution to `easy_infra`!

In order to streamline the review of your contribution we ask that you review and comply with the below requirements:
 - [X] Ensure that `make build` completes successfully
 - [X] Rebase your branch against the latest commit of the target branch, which likely should be `master`
 - [X] If there is an issue associated with your Pull Request, align your PR title with [this documentation](https://help.github.com/en/articles/closing-issues-using-keywords)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,32 +3,32 @@
 easy_infra is a docker container that simplifies and secures Infrastructure as Code deployments.
 
 ## Quickstart
-In order to run a command in the container, pass the command at the end of your `docker run` command and pass `docker` any needed files or variables, for instance:
-```
-docker run --rm -v $(pwd)/key:/keys/key -v $(pwd):/ansible seiso/easy_infra:latest "ansible-playbook -u user -i 192.0.2.230 --private-key /keys/key /ansible/playbook.yml"
+In order to run your code from within the container, volume mount your files into `/iac` and pass your command, such as `terrform validate`, as such:
```
Comment (pb-seiso) [inline README.md:6]: Typo - `terraform`

**Comment by pb-seiso** [comment]
Does it make more sense to have the README reference the local `easy_infra` rather than upstream `seiso/easy_infra`? As someone reading the git repo, it seems more implicit that I'd build and run the image locally rather than pulling it.

**Comment by pb-seiso** [comment]
@JonZeolla Aside from clarity on the README, all commands work as expected for me.

**Comment by JonZeolla** [comment]
I'd prefer `seiso/easy_infra` because I assume most people will be looking at `master` of the project and it will work more robustly; otherwise we'd need to prereq all of the commands with `make build` to be atomic.  I have seen other project update `README.md`s to pin to versions on version release commits and then revert to `latest` immediately afterwards which I'm not against if we can identify an automated workflow for it.

**Review by pb-seiso — APPROVED** [review]
LGTM!

**Pull request merged**